src/IW/Db/Issue.hs

```diff
@@ -4,48 +4,49 @@
 
 module IW.Db.Issue
        ( getIssues
-       , getIssuesByLabel
+       , getIssuesByLabels
        , upsertIssues
        ) where
 
-import IW.Core.Issue (Issue (..))
+import IW.Core.Issue (Issue (..), Label (..))
+import IW.Core.SqlArray (SqlArray (..))
 import IW.Core.WithId (WithId)
 import IW.Db.Functions (WithDb, executeMany, query, queryRaw)
 
 
--- | Returns all issues in the database
+-- | Returns all issues in the database.
 getIssues :: (WithDb env m) => m [WithId Issue]
 getIssues = queryRaw [sql|
     SELECT id, repo_owner, repo_name, number, title, body, labels
     FROM issues
     LIMIT 100
 |]
 
--- | Returns a list of issues filtered by label name
-getIssuesByLabel :: (WithDb env m) => Text -> m [WithId Issue]
-getIssuesByLabel = query [sql|
+-- | Returns all issues with at least one label in the given list.
```

Comment:
Documentation is clear! Though, it's an open question, how do we want to model filtering. For example, when a user enters labels `difficulty:easy` and `difficulty:medium` does he want to have issues that have both labels `easy` and `medium` or any of the labels? In this case, probably the first. But what if here enters `difficulty:easy` and `refactoring`? Maybe now he wants to have these two labels. We can't read the user's mind so we either will stick to one default or support both strategies. Not now, just something to think about.

Reply:
@chshersh Good question :thinking: